glusterd2/volume/store-utils.go

```diff
@@ -196,3 +196,12 @@ func Exists(name string) bool {
 
 	return resp.Count == 1
 }
+
```

Comment:
See earlier comment, this should be a method of the type and named `MetadataSize()`